.pre-commit-config.yaml

```
@@ -1,23 +1,41 @@
 repos:
   - repo: https://github.com/golangci/golangci-lint
-    rev: v1.27.0
+    rev: v1.50.1
     hooks:
       - id: golangci-lint
 
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v3.1.0
+    rev: v4.3.0
     hooks:
+      - id: check-json
       - id: check-merge-conflict
       - id: check-yaml
+      - id: detect-private-key
+      - id: pretty-format-json
+        args:
+          - --autofix
       - id: trailing-whitespace
+      - id: check-symlinks
+      - id: end-of-file-fixer
+      - id: mixed-line-ending
 
   - repo: https://github.com/igorshubovych/markdownlint-cli
-    rev: v0.23.1
+    rev: v0.32.2
     hooks:
       - id: markdownlint
 
+  - repo: https://github.com/executablebooks/mdformat
+    rev: 0.7.16
+    hooks:
+      - id: mdformat
+        additional_dependencies:
+          - mdformat-gfm
+          - mdformat-toc
+        # mdformat fights with terraform_docs
+        exclude: README.m(ark)?d(own)?
```

```suggestion
        # mdformat fights with terraform_docs
        exclude: README.m(ark)?d(own)?
```
No terraform here so I think we can let mdformat do all the things

---

You're correct. Pushed a fix.